### ISSUE TITLE
A bunch of issues relating to the menu, and menu header

### ISSUE THREAD

**Issue by KonradPZuse** [BeSite/jQuery.mmenu#274]
Hello all,

So far I really enjoy this menu, it does a lot, and seems to be better than most of the other menus I have found; however I have found a few issues which I'm sure can be fixed/I can be helped through them.

I have multiple images which I can provide if needed (all on my phone and my phone is not wanting to email).....

So anyways I will just explain each issue and how to reproduce, and hopefully we can get through this.

I have some issues (testing on Windows PC and Android), while my brother(testing on Mac and I phone) also has some issues(some might be the same I need to get his list which I will update later).

My code

  $(document).ready(function() {
      $("#menu").mmenu({
         header: {
             add:true,
             update:true,
             title: 'ProView'
             
         },
         footer:
         {
            add:true,
            update:true,
            title:'Version: 1.0.2'
         },
          offCanvas: {
            zposition : "front"
         },
      
         
         slidingSubmenus: false
         
      });
   });

1.1 I found that when enlarging the screen with either my Galaxy S3, as well as my computer that the menu header, as well as the menu got too large.  If possible it would be nice if they could be a fixed size so that I can only zoom into my content, while having my menu the same size.  Is there an extension/addon or something I can do to prevent this?

1.2  The next part of the issue seems to only happen on mobile.  If you zoom in, open the menu, and then try to zoom out, the menu gets cut off (this doesn't happen on desktop from what I see).  When you click out of the menu it messes up the header and the title by pushing it over to the left.  The more you are zoomed in  before exiting the menu, the closer the title will be to the menu button.

1.3  Also if zoomed out on mobile a little bit it breaks the header by having white space to the right. 

Also if in landscape mode, if you zoom out, then switch to portrait, the header is gone.  If switched back to landscape it's still there.  If switched back to portrait, it's still gone.  I tried then zooming in on landscape, and then switching back to portrait, which yielded a success in bringing the header back.  After the header was back I went back to landscape, and back to portrait, both were good.  From portrait I zoomed out then went to landscape, and then back to portrait which had most of the menu cut off.  I tried this before with portrait zoom, then landscape, back to portrait, and about half of the menu was cut off.  Zooming in seems to be what brings the menu back to life.  I tried to just zoom in with the half broken menu, then landscape, then portrait and it worked  so... weird :(.

1.4  I also noticed if zoomed out too far you cannot click the menu (this is not the case with desktop at fully zoomed out), something seems to break with mobile.

So basically If the menu/header gets fixed in size whenever there is a zoom it will solve all these issues I'm facing :)

--------------------------------------------------------

2.1  I have zpos to "front" but when that happens it covers the menu button.  Sometimes I can click the page to get off of the menu, but sometimes something on the page is preventing me from getting out of the menu. 

I basically have my menu set up as options, so if they click an option the menu doesn't close (if you remember I'm using an "a" and not "a href"", because the href will close the menu.    So basically it would be nice if you could include a close button for the menu with z-pos ="front" please :).

2.2 As for clicking on the page, the entire area is basically 1 giant SVG rectangle, where some parts of the page have a lot of SVG rectangles next to each other.  The area with many rectangles is where I couldn't click on; however the open areas I could (just where the large SVG rect was).  There isn't any onclick with these rectanges or anything.  I figured you might want this information incase this is why the menu isn't triggering a close, but maybe it's something else, so I at least wanted to give you some information!

--------------------------------

3.1  Not a bug, but I cannot seem to find a way to increase the text size of not only the menu items, but also the header title?

I'll update if I find anything more related or make a new thread... Thanks :)

**Comment by FrDH** [comment]
1:
It is recommended to disable zooming in and out using the "viewport" meta tag.
There is nothing I can do about the rendering issues of the device when rotating it.

2:
There's an issue that's soon to be fixed with the z-index when using `zposition: "front"`.

3:
You can override the CSS value for the `font-size` just like you normally would.

**Comment by KonradPZuse** [comment]
1.  Thanks I actually found that out after some research, I forgot to mention that in an edit.  I cannot stop the zoom out on my Samsung Galaxy s3 for some reason, but the Iphone I tried worked so >(.

2.  Okay thanks, I'll be awaiting :).  Is there a good way to know when updates come out, or just check periodically?

3.  Thanks, will do.


4.  I didn't re-read my post, but I also wanted to also ask something relating to this question https://github.com/BeSite/jQuery.mmenu/issues/259 where I ask about increasing the amount of text shown instead of an ellipsis.

I just put it in my code now to try it out, but that just seems to allow the text to flow through.

What I want to know now is what part of the css has to do with the size of the menu itself?  I want to experiment with how much the menu comes out, maybe change it for some mobile devices.

Thank you for all of the help, I appreciate it greatly!
